### PR TITLE
Plyr video loader with fallback 

### PR DESCRIPTION
**Pull request by nathakits**
- add plyr js and scss
- made new plyr js partial and added the partial to `baseof.html`
- add plyr shortcode

### CODE REVIEW

File: static/slate/stylesheets/_media.scss [hunk 1]
```diff
@@ -213,8 +213,9 @@
 }
 
 .content-container {
-  margin-right: 50%;
-  padding: 0 1.2em;
+  margin-right: $examples-width;;
```
Comment (willpatera) [inline static/slate/stylesheets/_media.scss]: extra semi-colon here. please fix @nathakits

File: static/slate/stylesheets/_variables.scss [hunk 2]
```diff
@@ -58,7 +58,7 @@ $lang-select-pressed-text: #fff !default; // color of language tab text when mou
 $nav-width: 230px !default; // width of the navbar
 $examples-width: 50% !default; // portion of the screen taken up by code examples
 $logo-margin: 20px !default; // margin between nav items and logo, ignored if search is active
-$main-padding: 28px !default; // padding to left and right of content & examples
+$main-padding: 100px !default; // padding to left and right of content & examples
```
Comment (willpatera) [inline static/slate/stylesheets/_variables.scss]: this seems like a lot of padding, no?

File: layouts/shortcodes/video-webm.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<div class="feature-content-container">
```
Comment (willpatera) [inline layouts/shortcodes/video-webm.html]: Add comment above the shortcode that documents the api/usage.

File: static/slate/stylesheets/_variables.scss (hunk 2 above)
Comment (nathakits) [inline static/slate/stylesheets/_variables.scss]: Looks like alot but on the website it looks a lot better than 28px

**Pull request merged**